Add documentation for config file settings (#4058)
opensearch-project/documentation-website: _ingest-pipelines/processors/ip2geo.md

| Key | Description | Default |
|--------------------|-------------|---------|

> | `plugins.geospatial.ip2geo.datasource.endpoint` | Default endpoint for creating the data source API. | Defaults to `https://geoip.maps.opensearch.org/v1/geolite2-city/manifest.json`. |

Reviewer: ```suggestion
| `plugins.geospatial.ip2geo.datasource.endpoint` | Default endpoint for creating the data source API. | Default is `https://geoip.maps.opensearch.org/v1/geolite2-city/manifest.json`. |
```

> | `plugins.geospatial.ip2geo.datasource.update_interval_in_days` | Default update interval for creating the data source API. | Defaults to 3. |

Reviewer: ```suggestion
| `plugins.geospatial.ip2geo.datasource.update_interval_in_days` | Default update interval for creating the data source API. | Default is 3. |
```

> | `plugins.geospatial.ip2geo.datasource.batch_size` | Maximum number of documents to ingest in a bulk request during the IP2Geo data source creation process. | Defaults to 10,000. |

Reviewer: ```suggestion
| `plugins.geospatial.ip2geo.datasource.batch_size` | Maximum number of documents to ingest in a bulk request during the IP2Geo data source creation process. | Default is 10,000. |
```
